rho_guess

```diff
@@ -0,0 +1,16 @@
+funct_list = []
```

Comment:
I haven't been able to test this because the functions and variables from the scf script are in separate directories, per your suggestion. To me, the big unknown is whether block_diag can use a list as an input. Please let me know of your suggestions.

reminder/fyi: https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.block_diag.html#scipy.linalg.block_diag

Reply:
According to the documentation you linked to `block_diag` accepts an arbitrary number of matrices to put on the diagonal, but it doesn't take them as a list. In Python you can unpack a list by putting a `*` in front of it, *i.e.*:

```.py
a_list = [1, 2, 3]

def a_fxn(arg0, arg1, arg2):
    print("Args: {} {} and {}".format(arg0, arg1, arg2))

a_fxn(*a_list) # Should print "Args: 1 2 and 3"
```

Reply:
As for the file issue, you really don't have a ton of functions so I recommend putting all of them in a single file say `my_fxns.py`

Reply:
Should I move the sdf files into an scf_guess subdirectory before attempting to execute the modified scf script?

Reply:
Yeah I would put the SDF files in a sub directory and make your script use them from there.